Adafruit_MQTT.h

```diff
@@ -107,7 +107,13 @@
 // Largest full packet we're able to send.
 // Need to be able to store at least ~90 chars for a connect packet with full
 // 23 char client ID.
+// Future TODO: This should be replaced by the ability to
```

Comment:
could be more informative ;)

Reply:
Ugh it got cut-off, why! Fixed in next commit.